File: Data/Models/Directory.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace Data.Models
{
    public partial class Directory
    {
        public Directory()
        {
            Contacts = new HashSet<Contact>();
        }

        public Guid Uuid { get; set; }
        public string Name { get; set; }
        public string Surname { get; set; }
        public string Company { get; set; }
        public bool IsActive { get; set; }

        public virtual ICollection<Contact> Contacts { get; set; }
    }
}

File: risetechmobile/lib/model/directory.dart
import 'package:risetechmobile/model/contact.dart';

class Directory {
  String uuid;
  String name;
  String surname;
  String company;
  bool isActive;
  List<Contact> contacts = new List<Contact>();

  Directory({
    this.uuid,
    this.name,
    this.surname,
    this.company,
    this.isActive,
  });

  factory Directory.fromJson(Map<String, dynamic> json) {
    return Directory(
      uuid: json["Uuid"],
      name: json["Name"],
      surname: json["Surname"],
      company: json["Company"],
      isActive: json["IsActive"],
    );
  }

  static List<Directory> fromJsonList(list) {
    if (list == null) return null;
    return list.map((item) => Directory.fromJson(item)).toList();
  }
}
